htdocs/website/index.php

```diff
@@ -4521,6 +4580,13 @@ function switchEditorOnline(forceenable)
 		print '</td></tr>';
 	}
 
+	print '<tr id="pageContent" class="hideobject"><td>';
+	print $langs->trans('PreviewPageContent');
+	print '</td><td>';
+	$doleditor = new DolEditor('content', GETPOST('content', 'restricthtmlallowunvalid'), '', 600, 'dolibarr_mailings', '', true, true, getDolGlobalInt('FCKEDITOR_ENABLE_MAILING'), 20, '90%');
```

Comment:
Use height of 200 instead of 600, we need a small preview but we won't edit the page content immediatly (hits is generaly done at next step), so no need for a high area.